src/open_inwoner/components/templates/components/Action/Actions.html

```diff
@@ -16,83 +16,83 @@
+                        <div class="action__label-title">
+                            {% trans "Status" %}:
+                        </div>
+                        {% if action|is_connected:request.user %}
+                            {% action_status_button action=action request=request plan=plan %}
+                        {% else %}
+                            <p class="action__label-value">
+                                {{ action.get_status_display }}
+                            </p>
+                        {% endif %}
+                    </section>
+                    <section class="action__label action__label--execute">
+                        <div class="action__label-title">{% trans "Uitvoeren voor" %}:</div>
+                        <p class="action__label-value">{% icon icon="calendar_today" outlined=True %} {{ action.end_date|date:"j F Y"|default:"-" }}</p>
+                    </section>
                 </div>
-                <div class="actions__label">
-                    <div class="table__explain">
-                        {% trans "Status" %}
-                    </div>
+                
+                <div class="action__actions">
                     {% if action|is_connected:request.user %}
-                        {% action_status_button action=action request=request plan=plan %}
-                    {% else %}
-                        {{ action.get_status_display }}
+                        {% get_action_edit_url action=action plan=plan as action_url %}
+                        {% button icon="edit" text=_("Bewerken") href=action_url icon_outlined=True transparent=True open_new_tab=True %}
+
+                        {% if plan %}
+                            {% url 'collaborate:plan_action_history' plan_uuid=plan.uuid uuid=action.uuid as action_history_url %}
+                            {% button icon="history" text=_("History") href=action_history_url uuid=action.uuid icon_outlined=True transparent=True %}
+                        {% else %}
+                            {% button icon="history" text=_("History") href="profile:action_history" uuid=action.uuid icon_outlined=True transparent=True %}
+                        {% endif %}
+
+                        {% get_action_delete_url action=action plan=plan as action_url %}
+                        {% render_form form=form method="POST" form_action=action_url extra_classes="confirm" spaceless=True data_confirm_title=_("Weet je het zeker dat je deze actie wilt verwijderen?") data_confirm_cancel=_("Nee, annuleren") data_confirm_default=_("Ja, verwijderen") %}
+                            {% csrf_token %}
+                            {% button icon="delete" text=_("Verwijderen") icon_outlined=True transparent=True %}
+                        {% endrender_form %}
                     {% endif %}
+                    {% button icon="file-pdf" text=_("Export to PDF") href="profile:action_export" uuid=action.uuid icon_outlined=True transparent=True %}
```

Comment:
The option to export individual actions is not in the design. 
The other export PDF button in the sidebar actually already exports all the actions.

So we need to ask the designer/product-owner if the PDF button is still needed - if that button is not needed, all functionality for that should be removed everywhere.

Reply:
The PDF button is visible (but doesn't work) when plan has expired:
 
<img width="890" alt="Screenshot 2025-05-26 at 12 35 57" src="https://github.com/user-attachments/assets/800ed421-52a9-4a5f-a1b1-d1f53bc82728" />